From 3f4af9d67adb530503e42ec10f39f2f6255df0ad Mon Sep 17 00:00:00 2001
From: James Cameron <quozl@laptop.org>
Date: Wed, 18 May 2016 12:00:33 +1000
Subject: [PATCH] Fix missing run, stop, clear icons

Three toolbar icons were not visible until pressed.

Cause was missing show() method call.
---
 pippy_app.py | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/pippy_app.py b/pippy_app.py
index fadf74f..55297bc 100644
--- a/pippy_app.py
+++ b/pippy_app.py
@@ -256,8 +256,10 @@ def initialize_display(self):
 
         icon_bw = Gtk.Image()
         icon_bw.set_from_file(os.path.join(icons_path, 'run_bw.svg'))
+        icon_bw.show()
         icon_color = Gtk.Image()
         icon_color.set_from_file(os.path.join(icons_path, 'run_color.svg'))
+        icon_color.show()
         button = ToolButton(label=_('Run!'))
         button.props.accelerator = _('<alt>r')
         button.set_icon_widget(icon_bw)
@@ -270,8 +272,10 @@ def initialize_display(self):
 
         icon_bw = Gtk.Image()
         icon_bw.set_from_file(os.path.join(icons_path, 'stopit_bw.svg'))
+        icon_bw.show()
         icon_color = Gtk.Image()
         icon_color.set_from_file(os.path.join(icons_path, 'stopit_color.svg'))
+        icon_color.show()
         button = ToolButton(label=_('Stop'))
         button.props.accelerator = _('<alt>s')
         button.set_icon_widget(icon_bw)
@@ -284,8 +288,10 @@ def initialize_display(self):
 
         icon_bw = Gtk.Image()
         icon_bw.set_from_file(os.path.join(icons_path, 'eraser_bw.svg'))
+        icon_bw.show()
         icon_color = Gtk.Image()
         icon_color.set_from_file(os.path.join(icons_path, 'eraser_color.svg'))
+        icon_color.show()
         button = ToolButton(label=_('Clear'))
         button.props.accelerator = _('<alt>c')
         button.set_icon_widget(icon_bw)
